src/app/products/product-card-rating/product-card-rating.component.html

```diff
@@ -0,0 +1,9 @@
+<ul
+          class="list-inline rating-list"
+          *ngFor="let star of stars"
+          style="display: inline-block"
+        >
+          <li [ngClass]="{ selected: star <= product.rating }">
```

Comment:
to make code more clear it's a good practice to wrap such conditions to method, for example:
          <li [ngClass]="{ selected: isStarLessOrEqualToRating(star) }">

and in .ts file you add new method:
`
public isStarLessOrEqualToRating(star: number): boolean { // so this method return true or false and with such naming it's more clear what this condition exactly means 
  return star <= this.product.rating; 
}
`